core/lib/soci/core/session.cpp

```diff
@@ -16,6 +16,8 @@
 #pragma warning(disable:4355)
 #endif
 
+#include <iostream>

```

Comment:
Why do we need `<iostream>` here?

Reply:
I used it for debug, I forgot to remove, nice catch !